src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuilding.java

```diff
@@ -1213,6 +1220,54 @@ public void setBeingGathered(final boolean gathering)
         this.beingGathered = gathering;
     }
 
+    /**
+     * Calculates the area of the building.
+     * @param world the world.
+     * @return the AxisAlignedBB.
+     */
+    public AxisAlignedBB getTargetableArea(final World world)
```

Comment:
can we move this into utils and have a wrapper method in building that only collects the needed data and passes it on?